### ISSUE TITLE
Improve inference speed of TTS 

### ISSUE THREAD

**Issue by sciai-ai** [espnet/espnet#3366]
I am working on improving the inference speed for TTS, I am already using GPUs. Can you please advise if ONNx is the right direction for this and whether  if it is possible to port the TTS (taco2, FS2) models to ONNX directly or will it need some underlying codes to be rewritten ?

Thanks

**Comment by sciai-ai** [comment]
Hi @kan-bayashi your kind advice will be very helpful. Thank you,

**Comment by sw005320** [comment]
> Can you please advise if ONNx is the right direction for this

I think so.
However, especially, FS2 is based on transformer and feed-forward networks, and I’m expecting that the benefit of ONNx would not be so large.

> if it is possible to port the TTS (taco2, FS2) models to ONNX directly

Yes, it should be straightforward (but I actually did not try, so I cannot them how it is direct, but at least it is easier than ASR)

Another interesting direction is quantization.
We got significant speed benefit in ASR https://github.com/espnet/espnet/pull/3210
